packages/ERTP/src/mathHelpers/inviteMathHelpers.js

```diff
@@ -27,16 +29,14 @@ const insistInviteExtent = inviteExtent => {
     passStyleOf(inviteExtent.instanceHandle) === 'presence' &&
       Object.entries(inviteExtent.instanceHandle).length === 0 &&
       Object.getPrototypeOf(inviteExtent.instanceHandle) === Object.prototype,
-    details`instanceHandle should be an empty object`,
+    `instanceHandle should be an empty object`,
```

Comment:
In general, when a simple string is used instead of a `details` template literal, I prefer quotes to backquotes, as you did on lines 13 and 22. I see why you didn't on line 17, but should here.

Reply:
Did we change our position on this? I prefer backquotes so I don't have to worry about embedded quotes.

Reply:
Do you prefer backquotes to quotes in general? If not, why here?

The reason I don't like backquotes here specifically is that this is a place where if there are substitution values (`${...}`) then there must usually also be a `details` template string tag. If this is already a tagless template literal, under maintenance it is too easy to add a substitution value but forget to add the `details` template literal tag. Everything will seem to continue to work just fine and all tests will pass. But you might be leaking info that you had not decided was ok to leak to callers.

Reply:
Yeah, I prefer backquotes to quotes in general. I just love not having to worry about embedded quotes and having to change them. 

Hmm, that concern makes sense. We can use `details` without needing string interpolation, right? What if I just always used `details`? I see it looks like I removed it here, but I think that was something about merging rather than a decision.

Reply:
```js
details`simple text`
```
is perfectly fine. The only reason to omit `details` is efficiency in the non-error case. This minor efficiency cost will rarely make a difference. So, yes, using `details` by default is fine.